t/02-rakudo/repl.t

```diff
@@ -6,13 +6,23 @@ use Test::Helpers;
 plan 45;
 
 my $*REPL-SCRUBBER = -> $_ is copy {
+    if $*DISTRO.is-win {
+    s/^^ "You may want to `zef install Readline` or `zef install Linenoise`"
+        " or use rlwrap for a line editor\n\n"//;
+    s/^^ "To exit type 'exit' or '^C'\n"//;
+    s:g/ ^^ "> "  //; # Strip out the prompts
+    s:g/    ">" $ //; # Strip out the final prompt
+    s:g/ ^^ "* "+ //; # Strip out the continuation-prompts
+    $_
+    } else {
```

Comment:
Same `^D` -> `^Z` note. But more important: the whole block is duplicate for the only purpose of changing one symbol in a message.

Reply:
Updated to `^Z`.
I tried using `if else` and `?? !!` within the block in order not to duplicate but tests kept failing.

Reply:
Try wrapping just that s/// line which contains the message. Or better use [`$(code)` construct](https://docs.perl6.org/language/regexes#Regex_interpolation)

Reply:
I just made the change. Is it ok now?